docs/HowTo/clusterconnectivity/aiven-kafka-cluster-ssl-protocol.md

```diff
@@ -50,7 +50,7 @@ Kafka® and Klaw using SSL protocol:
    icon that is available on the right-hand side of each cluster
    row.
 9. Open the `application.properties` file located in the
-   [klaw/cluster-api/src/main/resources] directory.
+   [klaw/cluster-API/src/main/resources] directory.
```

Comment:
This is a formatting error (from our side), it should be:

```suggestion
   `klaw/cluster-api/src/main/resources` directory.
```

as it is a file path. Then API also does not need to be all caps.